docs/proposals/04-new-core-gardener-cloud-apis.md

```diff
@@ -282,6 +283,41 @@ status:
   observedGeneration: 1
 ```
 
+### `Project` resource
+
+Special note: The `members` and `viewers` field of the `garden.sapcloud.io/v1beta1.Project` resource will be merged together into one `members` field.
+Every member will have a role that is either `admin` or `viewer`.
```

Comment:
#### comment

I would keep the role name `member`, not `admin`.
We might decide to narrow the privileges of the members(disallow creation/reading of secret/secretBindings for example and let only admins maintain cloud credentials).

But honestly, I have no strong opinion about this because we can introduce new roles latter that will have different access levels.

Reply:
Hm, valid suggestion/argument, but `member` role inside the `members[]` list is little confusing and not very explicit. I'd keep it as `admin`, because that's what's implemented today: Admins have full access (wrt to what is needed to interact with Gardener API) in their project.

Reply:
Yes, I agree.
We can say that previously we wrongly gave the name 'members' to users that has 'admin' privileges, so now it will be fixed - they will have the role `admin` and keep the same privileges.